examples/spawn-chain/README.md

```diff
@@ -37,3 +39,19 @@ cd www
 npm run start
 open http://localhost:8080
 ```
+
+### Get Demo Running on OS X
+
+If you don't already have the Lumen dev environment and just want to play with the demo, do these steps, ***then*** continue to the [link package](#link-package) steps.
+
+
+```
+brew uninstall rust
```

Comment:
was this meant to be `uninstall` ?

Reply:
@shaheenery is trying to get rid of the homebrew managed rust version and instead use the rustup managed version, so it means sense for that goal, but as I said, using `rustup` directly from the site's instructions is recommended.